scripts/commit-run-scripts/container_layers_count.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# Copyright 20122 EPAM Systems, Inc. (https://www.epam.com/)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+image=`docker ps --filter "id=$1" --format "{{.Image}}"`
+if [[ "$image" != "" ]]
```

Comment:
Let's add `else` with `exit 1` line, so that if image is not found for some case, command will fail.